queries/ecma/injections.scm

```diff
@@ -9,8 +9,14 @@
 (call_expression
   function: [
     (await_expression
-      (identifier)) @injection.language
+      [
+        (identifier) @injection.language
+        (member_expression
+          property: (property_identifier) @injection.language)
```

Comment:
This kindof open a weird can of wormholes. What about `foo.bar.sql`, `foo.bar.baz.sql`, ....

I'd prefer to keep this in my own config, but not in nvim-treesitter.

Unless you can give clear reasonings why this won't lead to the can of worms above

Reply:
The `foo.sql` pattern appears in one of the leading Javascript frameworks SQL SDKs (see [Vercel's PostgreSQL SDK](https://vercel.com/docs/storage/vercel-postgres/sdk#db)). This makes me feel that it is common enough to be covered here instead of in each user’s individual config (similar to how we already cover things like Styled Components via the `styled.div` injection).

On the other hand, I can’t think of a non user-constructed scenario (or any library) that would have a pattern like `foo.bar.sql`, `foo.bar.baz.sql`, `sql.bar`. This makes me feel that these are niche/rare enough to not be covered here but instead in a given user’s config.

While this doesn’t address why this wouldn’t open a can of worms, it does address why we might not feel compelled make any further changes related to it.

If you prefer the route of covering specific libraries (similar to Styled Components) let me know and I can amend this to have a (potentially separate) query only covering `sql` or even just `client.sql`.

Reply:
I will go for only covering for the most popular (like top 500 node packages).

Vercel's Postgres is popular enough so I'd give it a pass. Not sure about other cases though.

Reply:
Preferably writing a new query covering whatever that vercel postgres will be for + comments close to the new query